web/src/settings_org.ts

```diff
@@ -75,6 +75,7 @@ export function maybe_disable_widgets(): void {
         .prop("disabled", true);
 
     if (current_user.is_admin) {
+        $(".deactivate-realm-section").hide();
         $(".deactivate_realm_button").prop("disabled", true);
```

Comment:
You should remove this code for disabling and adding the tooltip now since we hide the complete section anyways.

Reply:
Also some code will be removed from `tippyjs.ts`, probably just a couple of lines, which is used to set tooltip for this particular button.